cpp/tests/framework/Engine.test.cpp

```
@@ -17,6 +22,13 @@ namespace robotick::test
 
 	ROBOTICK_REGISTER_WORKLOAD(TestSequencedGroupWorkload)
 
+	struct OverrunWorkload
+	{
+		void tick(const TickInfo&) { Thread::sleep_ms(10); }
```

Clang (14.0.6)

[warning] all parameters should be named in a function

(readability-named-parameter)